docs/index/index.rst

```diff
@@ -68,7 +68,7 @@ Daml Documentation
 
    Administration Introduction </index/operate-a-daml-ledger/administration-introduction/index>
    Canton Administration Quick Start </index/operate-a-daml-ledger/canton-administration-quickstart/index>
-   Monitoring Best Practices <https://www.canton.io/docs/dev/user-manual/usermanual/monitoring.html#status>
+   Monitoring Best Practices <canton/usermanual/monitoring.html#status>
```

Comment:
This unfortunately doesn't work. As far as I can find out, ToC entries must be either documents or external links; i.e. we can't link to a specific section of a document.

Reply:
If we structure the link as an external link even though it's to one of 'our' documents (eg https://docs.daml.com/canton/usermanual/security.html#moving-the-namespace-secret-key-to-offline-storage instead of /canton/usermanual/security.html#moving-the-namespace-secret-key-to-offline-storage) will that work?

Reply:
I think so, yes.